client-java/controller/src/main/java/org/evomaster/client/java/controller/internal/SutController.java

```diff
@@ -673,7 +677,7 @@ public RPCProblemDto extractRPCProblemDto(boolean isSutRunning){
         extractRPCSchema();
 
         Map<String, InterfaceSchema> rpcSchemas = getRPCSchema();
-        if (rpcSchemas == null || rpcSchemas.isEmpty()){
+        if (rpcSchemas.isEmpty()){
```

Comment:
prefer to keep the check for null, as cannot guarantee that `getRPCSchema` is always not null if its internal code is refactored